tokio-sync/src/mpsc/chan.rs

```diff
@@ -326,10 +326,13 @@ impl<T, S> Drop for Chan<T, S> {
     fn drop(&mut self) {
         use super::block::Read::Value;
 
+        // Safety: the only owner of the rx fields is Chan, and eing
```

Comment:
what is `eing`?

Reply:
```suggestion
        // Safety: the only owner of the rx fields is Chan, and being
```

I'm making my best guess....

Reply:
Yep, my 'b' key is dying.